sigle/src/stitches.config.ts

```diff
@@ -69,10 +69,14 @@ const greenDark = {
 };
 
 const red = {
```

Comment:
I've just added the colors being used in this pr to the ds - just wondering if this is fine or if it's better practice to add all of the green and red colors as they are now in the ds on figma?

Reply:
Should be good like this 👍